CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [0ver](https://0ver.org).
 
 - Configuration for OpenMetrics Encoding in metrics HTTP endpoint. Enabled by default but can be disable using `disable_openmetrics_encoding: true`.
 - Worker pool for projects polling: set `maximum_projects_poller_workers` with an integer value to control parallelism (defaults to `runtime.GOMAXPROCS(0)`)  
+- Augmented `disable_tls_verify` with `disable_health_check` additional parameter to drive the behaviour of checking healthiness of target service 
 - Configurable ServiceMonitor resource through the helm chart
 
```

Comment:
We should add a line to mention the polling of the CI variables as well.

Reply:
will add the proper text with regards to what we decide to do with gating

Reply:
updated, let me know what you think

Reply:
awesome thanks!